common/utils/src/main/resources/error/error-classes.json

```
@@ -1746,6 +1746,34 @@
     ],
     "sqlState" : "22012"
   },
+  "INVALID_AGGREGATE_FILTER" : {
+    "message" : [
+      "The FILTER expression <filterExpr> in an aggregate function is invalid."
+    ],
+    "subClass" : {
+      "CONTAINS_AGGREGATE" : {
+        "message" : [
+          "Expected a FILTER expression without aggregation, but found <aggExpr>."
+        ]
+      },
+      "CONTAINS_WINDOW_FUNCTION" : {
+        "message" : [
+          "Expected a FILTER expression without window function, but found <windowExpr>."
+        ]
+      },
+      "NON_DETERMINISTIC" : {
+        "message" : [
+          "Expected non-deterministic FILTER expression."
+        ]
+      },
+      "NOT_BOOLEAN" : {
+        "message" : [
+          "Expected FILTER expression of boolean type."
```

Let's follow the convention of `toSQLType`, and output the type in the upper case:
```suggestion
          "Expected a FILTER expression of the BOOLEAN type."
```

---

Thanks for the suggestions, I wasn't aware completely of this. Done.